interface/forms/reviewofs/new.php

```diff
@@ -28,390 +28,615 @@
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="vascular_surgery" name='vascular_surgery' />
+                                                <label class="form-check-label" for="vascular_surgery"><?php echo xlt('Vascular Surgery');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="cardiac_catheterization" name='cardiac_catheterization' />
+                                                <label class="form-check-label" for="cardiac_catheterization"><?php echo xlt('Cardiac Catheterization');?></label>
+                                            </div>
+                                        </div>
+                                        <div class="form-group">
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="coronary_artery_bypass" name='coronary_artery_bypass' />
+                                                <label class="form-check-label" for="coronary_artery_bypass"><?php echo xlt('Coronary Artery Bypass');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="heart_transplant" name='heart_transplant' />
+                                                <label class="form-check-label" for="heart_transplant"><?php echo xlt('Heart Transplant');?></label>
+                                            </div>
+                                            <div class="form-check">
+                                                <input type="checkbox" class="form-check-input" id="stress_test" name='stress_test' />
+                                                <label class="form-check-label" for="stress_test"><?php echo xlt('Stress Test');?></label>
+                                            </div>
+                                        </div>
+                                    </div>
                                 </div>
-                            </div>
                         </div>
-                        <div class="col-sm-3">
-                            <div class="form-group">
-                                <div class="checkbox">
-                                    <label>
-                                        <input type="checkbox" name='ulcerations'><?php echo xlt('Ulcerations');?>
-                                    </label>
+                    </fieldset>
+                    <fieldset>
+                        <legend><?php echo xlt('Gastrointestinal')?></legend>
```

Comment:
Look like in merge, that the following sections got duplicated:
Gastrointestinal
Genitourinary
Musculoskeletal
Endocrine
Additional Notes